amlip_cpp/test/unittest/node/statusNodeTest.cpp

```diff
@@ -141,17 +136,14 @@ TEST(StatusNodeTest, process_status_parent)
 
     for (const types::StatusDataType& data : data_arrived)
     {
-        if (data.id() == parent_id)
```

Comment:
I'd left the test as it was. If in the future we decide that we want status nodes to receive their own data, this have to change again